ENIGMAsystem/SHELL/Universal_System/Makefile

```diff
@@ -1,3 +1,6 @@
-SOURCES += $(wildcard Universal_System/*.cpp)
+SOURCES += $(wildcard Universal_System/*.cpp)\
+			$(wildcard Universal_System/Resources/*.cpp)\
+			$(wildcard Universal_System/Resources/Objects/*.cpp)\
+			$(wildcard Universal_System/Instances/*.cpp)
```

Comment:
@JoshDreamland @fundies how does this look? I couldn't find an example from any of our other Makefiles, so I don't know if I should write this differently? It works... seemingly.

Reply:
In the compiler / emake makefiles theres a wildcard thatll do a recursive search

Reply:
@fundies But how do you keep it from going into the `Extensions/` folder?

Reply:
well you could have the wildcard go up only one directory or you can do like you have now